### ISSUE TITLE
Feature: slow-cursor modifier key

### ISSUE THREAD

**Issue by papaGregor** [linearmouse/linearmouse#699]
### OS

macOS Sonoma 14.3

### LinearMouse

0.9.5

### Is your feature request related to a problem?

In the vein of the other Kensington-days-of-yore posts I've made today, there's another feature that was really useful for doing precision cursor movement, such as in Photoshop.

### Describe the solution you'd like

I'd like a modifier key that really slows down the cursor's movement compared to how much I'm moving the trackball (large trackball movement results in small cursor movement).

### Describe alternatives you've considered

Same as before: I haven't encountered this outside of Kensington's software on Mac OS 8

**Comment by EugeneTarasenko** [comment]
I agree. This would be a very helpful feature for situations where precise cursor movement is required. Personally, I would use it when adjusting parameter values in Capture One.

![goMLPaJHSYNWFJtA](https://github.com/user-attachments/assets/a60a7cc9-89cc-48aa-b33c-2334d5520220)
